hedera-node/docs/maintainers-guide.md

```diff
@@ -29,9 +33,20 @@ Follow [intellij-quickstart](intellij-quickstart.md) guide to run network on you
 Once the repository is opened in IntelliJ, to build the project from Intellij, open the Gradle tool window and
 run `Tasks/build/assemble` to on the root project. If you are using command line use `./gradlew assemble`.
 
-Start using either the Gradle command line
-`(./gradlew spotlessApply)`  or set up the [Google Java Format IntelliJ Plugin](https://github.com/google/google-java-format#intellij-android-studio-and-other-jetbrains-ides)
-in IntelliJ to format your code to follow Google Code Style.
+
+### Using gradle to ensure code quality
+
+To ensure your code meets project style standards, run the following before committing:
+
+* `./gradlew spotlessApply` - to automatically format code to the required style
+* `./gradlew checkAllModuleInfo` - iff you've changed any `module-info.java`
```

Comment:
Its okay to run this check whether or not any module descriptors have been changed. 

```suggestion
* `./gradlew checkAllModuleInfo` - if you've changed any `module-info.java`
```

Reply:
"Iff" <= math speak for "if and only if" but, yes, "if" fits too.  plus i know it's ok to run it regardless but it's not as fast as spotless.